README.md

```diff
@@ -26,22 +26,21 @@ Before you start, make sure that you have setup the supported version of Python
 
 | Input                  | Required                            | Description        |
 | ---------------------- | ----------------------------------- | ------------------ |
+| `token`                | ✅ Yes                              | GitHub token |
```

Comment:
#### 🥶 Not required (feel free to mark as resolve):

If available, please format the table; suggest using `markdownlint` or `Trunk` formatter.